src/components/App/App.js

```diff
@@ -35,6 +35,12 @@ class App extends Component {
     document.body.addEventListener('touchstart', this.resetTimeoutTimer);
     window.addEventListener('online', this.handleOnline);
     window.addEventListener('offline', this.handleOffline);
+    if (
+      !window.location.pathname.match(/\/admin.*/) &&
```

Comment:
Would using `startsWith` instead of regex read a little clearer?

```
const isValidRoute = location.pathname === '/' || location.pathname.startsWith('/admin')

if (!isValidRoute) {
  location.href = '/'
}
```

Reply:
Yep, totally forgot that was a feature.